### PR TITLE
Add CI workflow

### PR DESCRIPTION
**Pull request by mesaugat**
- Add CI workflow (install, lint, build and test)

There is no support for monorepos at the moment but there are workarounds similar to what we do in TravisCI.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install dependencies
+      working-directory: packages/db-model
```
Comment (mesaugat) [inline .github/workflows/ci.yml:29]: Hardcoding the path here because there is not a straight forward way to detect changes in monorepos.

Comment (kabirbaidhya) [reply .github/workflows/ci.yml:29]: :+1:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install dependencies
```
Comment (kabirbaidhya) [inline .github/workflows/ci.yml]:
```suggestion
    - name: Install
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      working-directory: packages/db-model
+      run: yarn
+
+    - name: Lint/Prettier check
```
Comment (kabirbaidhya) [inline .github/workflows/ci.yml]:
```suggestion
    - name: Lint
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+name: js-util CI
```
Comment (kabirbaidhya) [inline .github/workflows/ci.yml]:
```suggestion
name: CI
```

**Review by kabirbaidhya — APPROVED** [review]
Neat and clean. :ok_hand: 
image

**Pull request merged**